src/audio/codec_adapter/codec/cadence.c

```diff
@@ -171,7 +173,22 @@ static int apply_config(struct comp_dev *dev, enum codec_cfg_type type)
 		param = data;
 		comp_dbg(dev, "apply_config() applying param %d value %d",
 			 param->id, param->data[0]);
-		/* Set read parameter */
+		/* Update stream parameters */
```

Comment:
so the stream parameters will override config parameters, right? I think this is good!

Reply:
@dbaluta, correct :)

Reply:
Can we state this in the comments.